### ISSUE TITLE
Failed to sync action events with many attribute values

### ISSUE THREAD

**Issue by neo609** [MISP/MISP#3381]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Question
| OS version (server)       | ubuntu 16.04 LTS
| OS version (client)       | Windows 7
| PHP version               | 7.0.30
| MISP version / git hash   | 2.4.92
| Browser                   |  chrome

### Actual behavior
I have created the following five events on MISP instance(MISP_A) and tested the sync action between MISP Local Instances(MISP_A and MISP_B).

Sync Action:
MISP_A - (push) -> MISP_B

The Sync Action may have been successful, but some of the events with more than 10,000 attributes seem not to be able to completely sync.
Is there a limit on the maximum number of attribute values to sync?

### Steps to reproduce the behavior
■MISP_A:
Eventid1: #Attr. 10637
Eventid2: #Attr. 9731
Eventid3: #Attr. 29855
Eventid4: #Attr. 29843
Eventid5: #Attr. 4354

↓↓↓

■MISP_B:
Eventid1: #Attr. 9796
Eventid2: #Attr. 9731
Eventid3: #Attr. 11314
Eventid4: no sync
Eventid5: #Attr. 4354

### Logs, screenshots, configuration dump, ...
■MISP_A:
Jobs Message: Push to server 4 (= MISP_B) complete.
Logs: no Warnings and error

■MISP_B:
Jobs Message: no message
Logs: no Warnings and error

Best Reguards,

**Comment by Grem25** [comment]
Are you sure that all attributs from your MISP_A events are set with the distribution mode "connected"  or "all" ?

**Comment by neo609** [comment]
Thank you for your response!

Distribution mode of events is all "Connected" in MISP_A. And then, distribution mode of attributes is all "inherit".
I tried to check the status about MISP_B, I found the following error mesage in MISP/app/tmp/logs/error.log:

Error: Fatal Error (1): Allowed memory size of 134217728 bytes exhausted (tried to allocate 48250880 bytes) in [/var/www/MISP/app/View/Elements/eventattribute.ctp, line 216]
Error: [InternalErrorException] Internal Server Error

It seemed likely to be out of memory. So I modified to configuration file of MariaDB(tmp_table_size and max_heap_table_size) and php.ini(memory_limit).
After changing the settings, I could successflly synchronize.
I'm sorry that I failed to make sure.

Thank you,
